packages/resource-deployment/runtime-config/runtime-config.ci.json

```diff
@@ -0,0 +1,44 @@
+{
+    "logInConsole": {
+        "format": "Boolean",
+        "default": true,
+        "doc": "Property to decide if console logging is enabled"
+    },
+    "queueConfig": {
+        "maxQueueSize": {
+            "format": "int",
+            "default": 10,
+            "doc": "Maximum message the queue can have"
+        }
+    },
+    "taskConfig": {
+        "taskTimeoutInMinutes": {
+            "format": "int",
+            "default": 3,
+            "doc": "Timeout value after which the task has to be terminated"
```

Comment:
this is the schema type. the config json should be like this.

`{
    "logInConsole": true,
    "queueConfig": {
             "maxQueueSize": 100
     }
   ......
}
`